types/apidoc/index.d.ts

```diff
@@ -3,8 +3,15 @@
 // Definitions by: rigwild <https://github.com/rigwild>
```

Comment:
You may add your name there! :)
See https://github.com/lluia/DefinitelyTyped/blob/next-auth-types/types/next-auth/index.d.ts#L3-L4